packages/trader/src/Assets/SvgComponents/trade_explanations/img-vanilla.svg

```diff
@@ -0,0 +1 @@
+<svg width="272" height="147" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M268 0H4C1.79 0 0 1.779 0 3.973v139.054C0 145.221 1.79 147 4 147h264c2.209 0 4-1.779 4-3.973V3.973C272 1.779 270.209 0 268 0Z" fill="#fff"/><path d="M268 0H4C1.79 0 0 1.779 0 3.973v139.054C0 145.221 1.79 147 4 147h264c2.209 0 4-1.779 4-3.973V3.973C272 1.779 270.209 0 268 0Z" fill="#F2F3F4"/><path d="M157.12 89.6h32.77c3.22 0 5.83 2.404 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363H40.6c-3.21 0-5.82-2.393-5.82-5.363 0-2.96 2.6-5.364 5.83-5.364h64.08c3.22 0 5.83-2.403 5.83-5.363 0-2.96-2.6-5.364-5.83-5.364H55.16v-.05c-2.91-.337-5.1-2.621-5.1-5.323 0-2.702 2.19-4.986 5.1-5.324v-.04h20.4c3.21 0 5.82-2.403 5.82-5.363 0-2.97-2.6-5.373-5.83-5.373H48.61c-3.22 0-5.83-2.404-5.83-5.364 0-2.97 2.6-5.364 5.83-5.364h138.37c3.22 0 5.83 2.394 5.83 5.364 0 2.96-2.61 5.364-5.83 5.364h-53.9c-3.21 0-5.82 2.403-5.82 5.363 0 2.96 2.61 5.363 5.83 5.363h32.04c3.22 0 5.83 2.414 5.83 5.374 0 2.96-2.61 5.364-5.83 5.364h-17.48c-3.21 0-5.82 2.403-5.82 5.363 0 2.97 2.6 5.373 5.82 5.373h9.47Zm-126-42.937c3.23 0 5.84 2.403 5.84 5.363 0 2.98-2.61 5.373-5.83 5.373s-5.83-2.403-5.83-5.363c0-2.97 2.61-5.364 5.83-5.364l-.01-.01Zm136.92-21.454h40.79c3.21 0 5.82 2.393 5.82 5.363 0 2.97-2.6 5.363-5.82 5.363h-40.79c-3.21 0-5.82-2.403-5.82-5.363 0-2.97 2.6-5.363 5.82-5.363Zm-24.76 0c3.22 0 5.83 2.393 5.83 5.363 0 2.97-2.61 5.363-5.83 5.363s-5.82-2.403-5.82-5.363c0-2.97 2.6-5.363 5.82-5.363Zm70.64 21.454h10.2c3.22 0 5.83 2.403 5.83 5.363 0 2.98-2.61 5.373-5.83 5.373h-10.2c-3.21 0-5.82-2.403-5.82-5.363 0-2.97 2.6-5.364 5.82-5.364v-.01Zm-25.49 21.474h21.85c3.22 0 5.83 2.403 5.83 5.363 0 2.96-2.6 5.364-5.83 5.364h-21.84c-3.22 0-5.83-2.404-5.83-5.364s2.6-5.364 5.83-5.364h-.01Zm20.4 21.463h32.04c3.22 0 5.83 2.404 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363h-32.04c-3.22 0-5.83-2.393-5.83-5.363 0-2.96 2.61-5.364 5.83-5.364Zm-139.1 21.464h65.54c3.22 0 5.83 2.404 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363H69.73c-3.22 0-5.83-2.393-5.83-5.363 0-2.97 2.6-5.364 5.83-5.364Zm87.39 0h10.2c3.21 0 5.82 2.404 5.82 5.364 0 2.97-2.6 5.363-5.82 5.363h-10.2c-3.22 0-5.83-2.393-5.83-5.363 0-2.97 2.61-5.364 5.83-5.364Z" fill="#fff"/><g clip-path="url(#b)"><path d="M118 73.797h8v14.899h-8V73.797ZM84 96.145h42v7.45H76l8-7.45Z" fill="#85ACB0"/><path d="M112.343 51.45H106c-4.418 0-8-3.336-8-7.45h28v22.348h-8v-9.631l-50.344 46.878H62v-5.268l50.343-46.878Z" fill="#FF444F"/></g><g clip-path="url(#c)"><path d="M202 73.797h8V58.9h-8v14.898ZM168 51.45h42V44h-50l8 7.45Z" fill="#85ACB0"/><path d="M196.343 96.145H190c-4.418 0-8 3.336-8 7.45h28V81.247h-8v9.63L151.656 44H146v5.268l50.343 46.877Z" fill="#FF444F"/></g></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h272v147H0z"/></clipPath><clipPath id="b"><path fill="#fff" transform="translate(62 44)" d="M0 0h64v59.595H0z"/></clipPath><clipPath id="c"><path fill="#fff" transform="translate(146 44)" d="M0 0h64v59.595H0z"/></clipPath></defs></svg>
```

Comment:
Minify the icon.

Reply:
This file is already minified by `svgo`.